pkg/controller/petset/pet.go

```diff
@@ -165,20 +165,15 @@ type apiServerPetClient struct {
 }
 
 // Get gets the pet in the pcb from the apiserver.
-func (p *apiServerPetClient) Get(pet *pcb) (*pcb, bool, error) {
-	found := true
+func (p *apiServerPetClient) Get(pet *pcb) (*pcb, error) {
```

Comment:
why is it better to force the caller to deal with not found error? the method signature for this function before this change is very clear.